src/nvim/keycodes.c

```diff
@@ -184,6 +184,9 @@ static const struct key_name_entry {
   { K_KRIGHT,          "kRight" },
   { K_KRIGHT,          "KP6" },
 
+  { K_PS,              "PasteStart" },
```

Comment:
can we leave a hint about the intention of these keycodes, to emphasize that they are internal-only and not to be sent directly by UIs
```suggestion
  // Delimits pasted text (input from nvim_paste). Internal-only, not directly sent by UIs.
  { K_PS,              "PasteStart" },
```

Reply:
I can just remove these notations.